frontend/__tests__/src/fixtures.ts

```diff
@@ -434,7 +434,7 @@ export const EXPECTED_REPORT_VALUES = {
       valuesList: [
         {
           name: 'Failure rate',
-          value: '0%(0/2)',
+          value: '0.00%(0/2)',
```

Comment:
when to change `0.00%(0/2)` into 3 part(0.x, 0, 2)??
Change/design it as early as possible

Reply:
If the bug has been created, so no necessary to change it, makes no sense